### DISCUSSION TITLE
Installation Error (Samsung UN55J6520, T-HKMFAKUC-1560)

### DISCUSSION THREAD

**Discussion by Luis-BK201** [jhassan8/crunchyroll-tizen#107] [Q&A]
Greetings, I have this error, I was following the steps of a tutorial:

- You already got the ips.
- You are already in developer mode
- The bat is downloading public along with the rar.

What I can be doing wrong?

![Screenshot_1](https://github.com/jhassan8/crunchyroll-tizen/assets/77359323/7c19f8c0-d866-4e64-94f4-2bce2f4ca8cf)

**Comment by essayfr** [comment]
Did you ever find a solution to this?

**Reply by jhassan8** [reply]
Did you try to install it through tizen studio? Maybe there is a conflict with the tizen version of the TV and the .wgt

**Reply by essayfr** [reply]
how would I go about doing that?
